src/python/pants/backend/project_info/regex_lint.py

```diff
@@ -80,9 +80,7 @@ class RegexLintSubsystem(Subsystem):
 
         To activate this with the `lint` goal, you must set `[regex-lint].config`.
 
-        Unlike other linters, this can run on files not owned by targets, such as BUILD files. To
-        run on those, use `lint '**'` rather than `lint ::`, for example. We are exploring how to
-        improve this gotchas.
+        Unlike other linters, this can run on files not owned by targets, such as BUILD files.
```

Comment:
Some day this comment will be erased, because you will be able to run any linter on any file uncontroversially... :)